lib/Doctrine/ORM/Query/TreeWalker.php

```diff
@@ -335,7 +335,7 @@ public function walkInstanceOfExpression($instanceOfExpr);
     /**
      * Walks down a literal that represents an AST node, thereby generating the appropriate SQL.
      *
-     * @param mixed $literal
+     * @param AST\Literal $literal
```

Comment:
Passing anything else to the implementation in `SqlWalker` would cause an error because we access properties on `$listeral`.